cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -31,7 +31,7 @@ use assets_common::{
 	foreign_creators::ForeignCreators,
 	local_and_foreign_assets::{LocalFromLeft, TargetFromLeft},
 	matching::{FromNetwork, FromSiblingParachain},
-	AssetIdForTrustBackedAssetsConvert, MultiLocationForAssetId,
```

Comment:
@franciscoaguirre I am just thinking about removing usage of `MultiLocationForAssetId` from `polkadot-fellows/runtimes` for asset-hubs, and use there aleo exact `xcm::v3`, I think it could maybe help with easier bumping to the `polkadot-sdk`  release with XCMv4.